docs/README.md

```diff
@@ -37,4 +37,37 @@ hugo server -D
 
 to start a development server.
 
-**Important:** remember to revert the **config.toml** file when done.
+
+# Adding a page
+
+All components must have an info page that explains what the component is and how to use it and an
+interactive demo page that shows how the component looks and how it works.
+Most of the documentation files are automatically copied from the source.
+Run `node scripts/transfer-doc-files.js` in the root of the repo to copy all documentation files[^1].

```

Comment:
`npm run build:documentation`?

Reply:
build:documentation also builds the components. Added a note to use it if they need to rebuild the components.